From aee1b4427f75e40e575e3d93ab8309ebae5d53ed Mon Sep 17 00:00:00 2001
From: "Qi, Yadong" <yadong.qi@intel.com>
Date: Fri, 23 Feb 2018 14:12:07 +0800
Subject: [PATCH 0554/1240] trusty: Update dependency of trusty module

Trusty is supported only for x86_64 arch.
Modify Kconfig to make it depends on x86_64.

Change-Id: Ia52a8ba05f2de3d423e070a53e7368901b20ada7
Signed-off-by: Qi, Yadong <yadong.qi@intel.com>
Signed-off-by: Zhou Furong <furong.zhou@intel.com>
---
 drivers/trusty/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/trusty/Kconfig b/drivers/trusty/Kconfig
index 7d26922..a230dad 100644
--- a/drivers/trusty/Kconfig
+++ b/drivers/trusty/Kconfig
@@ -6,7 +6,7 @@ menu "Trusty"
 
 config TRUSTY
 	tristate "Trusty"
-	depends on X86
+	depends on X86_64
 	default n
 
 config TRUSTY_FIQ
-- 
2.7.4

